### ISSUE TITLE
Return added data and removed data as hash

### ISSUE THREAD

**Issue by Nowaker** [bpohoriletz/hash_deep_diff#11]
It would be nice to have a way to return added data (hash elements that would be presented as +, or -+ with the + value returned) and removed data as hash (hash elements that would be presented as -, or -+ with the - value returned). It should return a hash with all nesting preserved.

**Comment by bpohoriletz** [comment]
Hello,

sth like

```ruby
  left = { a: { b: c } }
  right = { a: { b: :d }, { e: :f } }
  HashSepDiff::Comparison.new(left, right).additions
  =>  { a: { b: :d}, { e: :f } }
  HashSepDiff::Comparison.new(left right).deletions
  =>  { a: { b: :c } }
```
or sth else?

**Comment by bpohoriletz** [comment]
ok, will do =)

**Comment by bpohoriletz** [comment]
@Nowaker I have the first version ready in dev branch. I will do few refactoring and include it in 0.8.0. If you'd like to check it earlier here are [tests](https://github.com/bpohoriletz/hash_deep_diff/tree/dev/test/integration/locales/yml)

**Comment by bpohoriletz** [comment]
@Nowaker implemented in 0.8.0 https://github.com/bpohoriletz/hash_deep_diff#raw-report
